tutorials/introductory/customizing.py

```diff
@@ -1,24 +1,43 @@
 """
 .. redirect-from:: /users/customizing
 
+=====================================================
 Customizing Matplotlib with style sheets and rcParams
 =====================================================
 
 Tips for customizing the properties and default styles of Matplotlib.
+There are three ways to customize Matplotlib:
+
+    1. :ref:`Using style sheets<customizing-with-style-sheets>`.
+    2. :ref:`Setting rcParams at runtime<customizing-with-dynamic-rc-settings>`.
+    3. :ref:`Changing your matplotlibrc file<customizing-with-matplotlibrc-files>`.
+
+Style sheets take precedence over :file:`matplotlibrc` files and setting
+rcParams at runtime takes precedence over style sheets.
```

Comment:
These should probably be listed in priority order.

Reply:
I was also thinking about this, but finally mirrored the order of the tutorial sections. If you prefer, I can list the items in priority order. When doing this, should I also change the section order accordingly or keep it as is?

Reply:
It probably should be, but I'm not too sure if high to low or vice versa.

Reply:
I reordered the list and the section from high to low priority.